dapps/marketplace/src/pages/transaction/Transaction.js

```diff
@@ -168,23 +176,24 @@ const Transaction = props => {
                 <>
                   {Progress}
                   {Offer}
+                  {VerticalSeparator}
                   {Escrow}
-                  {Listing}
+                  {VerticalSeparator}
                   {About}
+                  {VerticalSeparator}
                   {History}
                 </>
               ) : (
                 <div className="row">
-                  <div className="col-md-7 col-lg-8">
-                    {Progress}
-                    {History}
-                    {Listing}
-                  </div>
-                  <div className="col-md-5 col-lg-4 side-bar">
-                    {Offer}
-                    {Escrow}
-                    {About}
+                  <div className="col-12">{Progress}</div>
+                  <div className="col-12 d-flex">
```

Comment:
maybe we can just use a `<div></div>` instead of `<div className="row"><div className="col-12'></div></div>`?

Reply:
Not sure I follow... are you suggesting removing `row` and `col-12` classes.

Reply:
Yea, a row + col-12 is the same as just a div, right?

Reply:
Yup good point 👍